js/browser.js

```diff
@@ -4,32 +4,35 @@ var Billing = Class(Emitter, function (supr) {
 	this.purchase = function (item) {
```

Comment:
Can you convert this file to 2 spaces. Other files in this project use 2 spaces

Reply:
Okay.